plugins/spz/io/SpzWriter.cpp

```diff
@@ -33,6 +33,7 @@
 ****************************************************************************/
 
 #include "SpzWriter.hpp"
+#include "Util.hpp"
```

Comment:
Can we rename this file to `SpzUtil.hpp` or `private/spz/Util.hpp`? It is going to be hard to Ctrl-P for with that name.